Phase01Project1/src/SearchFilters/OrSearchFilter.java

```diff
@@ -0,0 +1,33 @@
+package SearchFilters;
+
+import java.util.ArrayList;
+import java.util.Map;
+import Documents.Document;
+
+public class OrSearchFilter extends SearchFilter {
+
+    public OrSearchFilter(ArrayList<String> filterWords, Map<String, ArrayList<Document>> wordDocuments, ArrayList<Document> resultDocuments){
+        super(filterWords, wordDocuments, resultDocuments);
+    }
+    @Override
+    public ArrayList<Document> handleFilterWords() {
```

Comment:
sorry man.
Duplicate code with NotSearchFilter. 
i suggest using super method instead of creating another one.